apps/api/src/server/appeals/case-officer/case-officer.controller.js

```diff
@@ -93,11 +107,13 @@ export const confirmLPAQuestionnaire = async (request, response) => {
 };
 
 /**
+ * @param {import('express').Request} request
```

Comment:
list of params rather than @type(requestHandler)

Reply:
what do you mean?